packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
Ok so I can't add 0.1-0.3 .tbz to github since I didn't do proper releases from 0.1-0.3, I only tagged. Mindy did it properly for 0.4 and 0.5, so I added charrua-core-0.4.tbz to github at least. Never mind me, carry on :D